src/main/java/li/cil/tis3d/common/block/CasingBlock.java

```diff
@@ -149,26 +151,26 @@ public boolean activate(final BlockState state, final World world, final BlockPo
                             }
                         }
                     }
-                    return true;
+                    return ActionResult.SUCCESS; // XXX
```

Comment:
Sorry about that, the `XXX` is because I didn't know which of the 4 `ActionResult`s is appropriate here and guessed. From yarn:
```
/**
    * Indicates an action is performed and the actor's hand should swing to
    * indicate the performance.
    */
   SUCCESS,
   /**
    * Indicates an action is performed but no animation should accompany the
    * performance.
    */
   CONSUME,
   /**
    * Indicates an action is not performed but allows other actions to
    * perform.
    */
   PASS,
   /**
    * Indicates that an action is not performed and prevents other actions
    * from performing.
    */
   FAIL;
```

What I should have done is observe the behavior / hand motion in 1.14 and make sure it is replicated (will do that).

Reply:
Alright that was foolish, `SUCCESS` is clearly the only return value that replicates the hand swinging.